lib/ab/missing_variant.rb

```diff
@@ -0,0 +1,28 @@
+module Ab
+  module MissingVariant
+    def method_missing(meth, *args, &block)
+      if variant_method?(meth)
+        log_missing_variant(meth)
+        false
+      else
+        super
+      end
+    end
+
+    def respond_to_missing?(meth, *)
+      variant_method?(meth) ? true : super
+    end
+
+    private
+
+    def variant_method?(meth)
+      meth.to_s.end_with?('?')
+    end
+
+    def log_missing_variant(meth)
+      return unless self.class.respond_to?(:logger) && self.class.logger
+
+      self.class.logger.debug("[AB_testing] Checking non-existing variant: #{meth}")
```

Comment:
I think logging is usually done to a simple `Logger.new(STDOUT)` by default, or a simple stub.